common/org.testeditor.dsl.common.testing/pom.xml

```diff
@@ -12,7 +12,7 @@
 	</parent>
 
 	<artifactId>org.testeditor.dsl.common.testing</artifactId>
-	<packaging>eclipse-plugin</packaging>
+	<packaging>eclipse-test-plugin</packaging>
```

Comment:
The project `org.testeditor.dsl.common.testing` was not intended to be used as a test plugin. Its intention was to provide common code for tests. We should create a new project `org.testeditor.dsl.common.tests` for tests of the common bundle.